src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -12,8 +12,23 @@ export class SavedObjectsClient {
     this._callAdminCluster = callAdminCluster;
   }
 
-  async create(type, body = {}) {
-    const response = await this._withKibanaIndex('index', { type, body });
+  /**
+   * Persists an object
+   *
+   * @param {string} type
+   * @param {object} body - { attributes: {}, id: myId }
+   * @param {object} options
+   * @param {boolean} options.overwrite - defaults to false
+   * @returns {promise}
+  */
+  async create(type, body = {}, options = {}) {
```

Comment:
[options now accepts an id](https://github.com/tylersmalley/kibana/blob/5d00b4f55ece953ec2372027d244b00c2101a43d/src/server/saved_objects/client/saved_objects_client.js#L21). Let me know if this works for you